packages/calcite-design-tokens/support/tests/__snapshots__/index.spec.ts.snap

```diff
@@ -840,431 +745,14 @@ export const calciteZIndex9 = "900";
 exports[`generated tokens ES6 global should match 1`] = `
 "/
 
-export const calciteBorderWidthNone = "0";
-export const calciteBorderWidthSm = "1px";
-export const calciteBorderWidthMd = "2px";
-export const calciteBorderWidthLg = "4px";
-export const calciteColorBackground = {"light":"#f8f8f8","dark":"#353535"};
-export const calciteColorBackgroundNone = "rgba(255, 255, 255, 0)";
```

Comment:
Also, can you update the snapshot to cover semantic tokens? These are no longer present.